src/container/Movies.js

```diff
@@ -1,14 +1,53 @@
 import React,{Component} from "react"
 import axios from 'axios'
+import Moviescss from "./Moviescss.css"
 import Movie_list from "../components/Movie_list"
+import Spinner from '../common/Spinner'
+import MovieFilters from '../components/MovieFilters'
 
 class Movies extends Component{
 	constructor(props){
 		super(props)
 	this.state={
-		movies_list:[]
+		movies_list:[],
+		search_value:'',
+		message: '',
+		isloading: false,
+		filter:{
+			is4: false,
+			is3: false,
+			is2: false
+		}
+			
 	}
 	this.handleDelete = this.handleDelete.bind(this)	
+	this.searchmovie = this.searchmovie.bind(this)
+	this.handleChange = this.handleChange.bind(this)
+	this.filter = this.filter.bind(this)
+	}
+	searchmovie(e){
+	
+		var search_list = this.state.movies_list.filter(
+			(movie)=> (
+				movie.name == this.state.search_value.search
+			));
+		this.setState({
+			movies_list : search_list
+		});
+		if(search_list.length > 0){
+			console.log('serched')
```

Comment:
Log statements are useful for  debugging the errors. Add them only if they are really required.